src/analytics/api/models/base/data_processing.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, request, jsonify
+from datetime import datetime
+import logging
+from api.utils.pollutants.report import (fetch_air_quality_data, 
+                                         pm_by_city,pm_by_country,
+                                         pm_by_region, monthly_mean_pm_site_name,
+                                         datetime_pm2_5, mean_pm2_5_by_month_year, 
+                                         mean_pm2_5_by_month_name, query_bigquery, 
+                                         mean_pm2_5_by_month, results_to_dataframe,
+                                           mean_pm2_5_by_year, mean_daily_pm2_5, 
+                                           mean_pm2_5_by_site_name, mean_pm2_5_by_hour)
+ 
+
+app = Flask(__name__)
```

Comment:
Thanks @wabinyai for this.
Just a quick clarification, do we need to start a new instance of the Flask App?

Reply:
thanks for the keen eyes